### DISCUSSION TITLE
Workflows | Filter conditions | Assigned to

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17108] [Feature improvement]
Workflows doesn't have filter conditions "Assigned to" is empty. This condition will be helpful in import process when we don't want overwrite "Assigned to" in existing record. To preserve "Assigned to" imported file doesn't have filled "Assigned to" field, but in this case new records are assigned to no one. Above filter condition will allow to fix it automatically

best

**Comment by rskrzypczak** [comment]
What you're describing doesn't really make sense.
The "Assigned to" field cannot be left blank when creating a record.
Even if you don't fill it in, the system should complete it itself, unless you somehow import data with your own scripts in a non-standard way.
